waveguide_mode_converter/mode_converter_meep_opt.py

```diff
@@ -37,13 +39,14 @@
 wvls = (1.265, 1.270, 1.275, 1.285, 1.290, 1.295)
 frqs = [1/wvl for wvl in wvls]
 
-minimum_length = 0.09  # minimum length scale (μm)
+minimum_length = 0.05  # minimum length scale (μm)
```

Comment:
Did you drop the minimum feature size because this is what's done in the other paper and you want to be consistent? You should still get great results with 90 nm.